server/src/test/resources/csars/yaml/valid/lamp-csar/scripts/start_myphpapp.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+sudo mv myphpapp.php /var/www/html/
```

Comment:
This is more a `create_myphpapp.sh` script.

Reply:
but with this action you start the php application. However i think in the test-csar it doesnt matter

Reply:
you don't start it with moving it but nvm